docs/resources/scenarios/sample-scenario-overview.md

```diff
@@ -0,0 +1,20 @@
+---
+title: 'Sample Scenarios for Office Scripts'
```

Comment:
Update casing in article title:  Sample Scenarios for Office Scripts -> Sample scenarios for Office Scripts

Also -- would it make sense to make the titling of this article consistent with that of the [Sample scripts](https://review.docs.microsoft.com/en-us/office/dev/scripts/resources/excel-samples?branch=AlexJ-Samples) article -- i.e., `Sample scenarios for Office Scripts in Excel on the web (preview)`